File: app/src/main/java/com/cyl/musiclake/bean/FolderInfo.java
package com.cyl.musiclake.bean;

import org.litepal.crud.LitePalSupport;

public class FolderInfo extends LitePalSupport {

    public final String folderName;
    public final String folderPath;
    public final int songCount;

    public FolderInfo() {
        this.folderName = "";
        this.folderPath = "";
        this.songCount = -1;
    }

    public FolderInfo(String _folderName, String _folderPath, int _songCount) {
        this.folderName = _folderName;
        this.folderPath = _folderPath;
        this.songCount = _songCount;
    }
}

File: app/src/main/java/com/cyl/musiclake/bean/FolderInfo.kt
package com.cyl.musiclake.bean

import org.litepal.crud.LitePalSupport

class FolderInfo : LitePalSupport {

    val folderName: String
    val folderPath: String
    var songCount: Int

    constructor(_folderName: String, _folderPath: String, _songCount: Int) {
        this.folderName = _folderName
        this.folderPath = _folderPath
        this.songCount = _songCount
    }
}
